tests/publiccloud/run_ltp.pm

```diff
@@ -141,19 +156,47 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    my $cmd = 'perl -I runltp-ng runltp-ng/runltp-ng ';
-    $cmd .= '--logname=ltp_log --verbose ';
-    $cmd .= '--timeout=1200 ';
-    $cmd .= '--run ' . get_required_var('LTP_COMMAND_FILE') . ' ';
-    $cmd .= '--exclude \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
-    $cmd .= '--backend=ssh';
-    $cmd .= ':user=' . $instance->username;
-    $cmd .= ':key_file=' . $instance->ssh_key;
-    $cmd .= ':host=' . $instance->public_ip;
-    $cmd .= ':reset_command=\'' . $reset_cmd . '\'';
-    $cmd .= ':ssh_opts=\'-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no\' ';
-    $cmd .= '--json_filter=openqa ';
-    assert_script_run($cmd, timeout => get_var('LTP_TIMEOUT', 30 * 60));
+    my $ltp_timeout = get_var('LTP_TIMEOUT', 30 * 60);
+    my $ltp_suite_timeout = 1200;
+    my $cmd;
+  
+    if ( $ltp_runtime =~ /^python/i ) {
+        zypper_call("in -y python3-paramiko python3-scp");
+
+        my $sut= '';
+        $sut .= ':user=' . $instance->username;
+        $sut .= ':sudo=1';
+        $sut .= ':key_file=' . $instance->ssh_key;
+        $sut .= ':host=' . $instance->public_ip;
+        $sut .= ':reset_command=\'' . $reset_cmd . '\'';
+        $sut .= ':hostkey_policy=missing';
+        $sut .= ':known_hosts=/dev/null';
+
+        $cmd = 'python3 runltp-ng/runltp-ng ';
+        $cmd .= "--json-report=$root_dir/result.json ";
+        $cmd .= '--verbose ';
+        $cmd .= '--run-suite ' . get_required_var('LTP_COMMAND_FILE') . ' ';
+        $cmd .= '--skip-tests \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
+        $cmd .= '--sut=ssh' . $sut . ' ';
+    } else {
```

Comment:
```suggestion
    } elsif ( $ltp_runtime eq 'perl') {
```

This is prone to use errors when extending the test cases. One should only act on the acceptable input and reject other methods to avoid user confusion.

Here I could just e.g. create a test run with `LTP_RUNTIME=pythn` and it would happily run the perl codebase, while it should fail and tell me that I have a typo in my test settings.

Reply:
I can agree on that but in such a case we also need 
```
} else {
die ('Not supported LTP_RUNTIME provided'); 
}
```

Reply:
Sorry for my reply, but the switching context should be temporary, while default is the only perl one, until we consolidate a stable python context, that will remain the only one, removing all these options.

Reply:
please check https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15583#discussion_r988028961 if ok